test/test.googleauth.ts

```diff
@@ -2340,6 +2340,18 @@ describe('googleauth', () => {
         assert.deepStrictEqual(res.data, data);
         scopes.forEach(s => s.done());
       });
+
+      it('getCredentials() should return the service account email for external accounts', async () => {
```

Comment:
Nit, You could do:

```
describe('getCredentials()', () => {
  it('should return the service account email for external accounts', async () => 
  {
        // Set up a mock to return path to a valid credentials file.
        const email = saEmail;
        const configWithImpersonation = createExternalAccountJSON();
        configWithImpersonation.service_account_impersonation_url =
          getServiceAccountImpersonationUrl();
        const auth = new GoogleAuth({credentials: configWithImpersonation});
        const body = await auth.getCredentials();
        assert.notStrictEqual(null, body);
        assert.strictEqual(email, body.client_email);
      });
});
```

So that tests read a bit better.